tests/test_file_readers.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import numpy as np
+
+import file_readers as rd
+
+
+def test_preprocess():
+    """
+    Tests preprocess function by asserting title,
+    shape of corpus, and correct line reading.
+    """
+    txt_path = "tests/data/test.txt"
+    file = open(txt_path)
+    corpus = np.array(rd.preprocess(file), dtype=object)
+    file.close()
+
+    assert np.shape(corpus) == (26,)
+    assert corpus[0] == 'Testing Text File'
+    assert corpus[2] == 'Link to generator repo!'
+    assert corpus[20] == 'Here are some Chinese characters: .'
+    assert corpus[22] == 'The vowels: are , , , , , , .'
+    assert corpus[24] == 'We can also test for mathematical symbols: , , , , , X, %,  ,a, , , +, = ,-.'
+    assert corpus[25] == 'Finally, here are some emoticons: .'
```

Comment:
Do you need all of these asserts? The text is coming from the same file. Why not just test equality on the entire file? This is neither memory nor time intensive for a file that small.

Reply:
Created a txt file to check equality, rather than all these asserts.